histomicsui/web_client/stylesheets/panels/metadataWidget.styl

```diff
@@ -0,0 +1,104 @@
+.h-metadata-widget
+  .h-metadata-container
+    background-color #f0f0f0
+    color #333
+    font-size 14px
+    font-weight bold
+
+  .s-panel-title-container
+    .s-no-panel-toggle
+      position relative
+
+  .g-widget-metadata-header
+    background-color #f0f0f0
+    color #333
+    font-size 14px
+    font-weight bold
+
+  .g-widget-metadata-header-key-value
+    margin-left 5px
+    font-size 10px
+    text-overflow ellipsis
+    overflow hidden
+    max-width 100px
+    display inline-block
+    white-space nowrap
+    margin-bottom 2px
+
+  .g-widget-metadata-row
+    margin 3px
+    margin-top 5px
+    border-bottom 1px solid #f0f0f0
+    min-height 25px
+    font-size 14px
+    color #777
+
+  .g-widget-metadata-key, .g-widget-metadata-value
+    margin 5px
+    display inline-block
+    overflow hidden
+    text-overflow ellipsis
+
+  .g-widget-metadata-key
+    font-weight bold
+    width 30%
+
+    span.small
+      float right
+
+  .g-widget-metadata-value
+    width 100%
+
+    pre
+      padding 0
+      font-size 14px
+      color #777
+      background-color transparent
+      border none
+      max-height 320px
+      overflow auto
+      margin-bottom 0
+
+  .g-json-editor
+    width 54%
+    height 400px
+    display inline-block
+    background-color white
+    vertical-align middle
+    margin-left 5px
+    margin-right 5px
+
+    .ace_editor, textarea
+      height 365px !important
+
+  button.g-widget-metadata-add-button
+    float none
+    margin 0 5px 8px 0
```

Comment:
Somehow with the `8px` value, the arrow marker that indicates whether the panel is open or closed is pushed down.  This looks better to me a `0 5px 0 0`.

Reply:
Thank you, David. I have changed it.